addons/hr_expense/models/hr_expense.py

```diff
@@ -1017,6 +1115,14 @@ def action_sheet_move_create(self):
         self.activity_update()
         return res
 
+    def action_unpost(self):
+        for sheet in self:
+            move = sheet.account_move_id
+            sheet.account_move_id = False
+            move.button_draft()
+            move.button_cancel()
```

Comment:
Cancelling a move might not be the best idea/practice. It will create holes in the sequence.

Reply:
```suggestion
            move._reverse_moves(cancel=True)